distribution/src/config/jvm.options

```diff
@@ -38,13 +38,16 @@
 8-13:-XX:+UseCMSInitiatingOccupancyOnly
 
 ## G1GC Configuration
-# to use G1GC, uncomment the next two lines and update the version on the
-# following three lines to your version of the JDK
+# to use G1GC (ParallelGC for heaps smaller than 4G),
+# uncomment the next three lines and update the version on the
+# following two lines to your version of the JDK
 # 8-13:-XX:-UseConcMarkSweepGC
 # 8-13:-XX:-UseCMSInitiatingOccupancyOnly
-14-:-XX:+UseG1GC
 14-:-XX:G1ReservePercent=25
```

Comment:
won't these G1 options break the system if parallelGC is selected?

Reply:
with -XX:+UseParallelGC java seem to ignore -XX:G1ReservePercent=25 and XX:InitiatingHeapOccupancyPercent  them as long as -XX:+UseG1GC is not specified. On the other hand it might be confusing to have G1GC options there when running ParallelGC.

Reply:
Have you tested if that is true for all G1 options? I wonder if we should completely move all GC options into ergonomics.